packages/clients/afm/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.0.1
 
 - Fix: The included example files have been updated to the new syntax and work again.
+- Fix: Remove configurable attribute windowLength which does not exist anymore.
```

Comment:
Changelog entry is not necessary, as the respective customer does not recognize that parameter.